p2p/validation/MESSAGE_VALIDATION_CONSENSUS.MD

```diff
@@ -0,0 +1,455 @@
+# Message Validation on Consensus messages
+
+## Summary
+
+This is a specification for the networking layer's message validation done on *QBFT* messages and *Decided* messages (aggregation of a quorum of commit messages). It is done by utilizing LibP2P's [extended validators](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#extended-validators).
+The main goal of this validation is to prevent DOS attacks. In order to achieve this goal we apply some synchronicity assumptions to our communication model. We can do this because QBFT is partially synchronous, and Ethereum applies synchronicity assumptions as well.
```

Comment:
we do not validate some of the message? like signature?

Reply:
Near the end of the document we define how to do aggregate signature

Reply:
seems the way you approach aggregation might delay messages no?

Reply:
We will conclude the issue in this comment thread:
https://github.com/bloxapp/ssv-spec/pull/188#discussion_r1174344529